### PR TITLE
[Documentation:Developer] Account for changed name of fork

### PR DESCRIPTION
**Pull request by micpap25**
This commit acknowledges external users may have changed the name of their fork from "Submitty" and guides developers on finding and using the fork name correctly

### CODE REVIEW

File: _docs/developer/getting_started/commit_to_PR_from_fork.md [hunk 1]
```diff
@@ -22,49 +22,52 @@ The instructions below are for command line use of git.
...
     ```
 
+    Additionally, find the name of the fork
+    on the fork's page on Github.
+    
 
-2.  Next, make a local branch on your computer in your working repository
-    with the proposed code changes.  Here are the command lines
-    (substitute the fork & branch name we found above):
+3.  Next, make a local branch on your computer in your working repository
```
Comment (bmcutler) [inline _docs/developer/getting_started/commit_to_PR_from_fork.md:34]: why did you change this number to 3? 
Did you mean to insert a step 2 above?

Comment (micpap25) [reply _docs/developer/getting_started/commit_to_PR_from_fork.md]: Sorry about that; the file already had issues with the numbering which i passed through and fixed now

**Review by bmcutler — APPROVED** [review]
thanks for the update!
